app/services/google_credentials_generator.rb

```diff
@@ -67,9 +66,9 @@ def substitute_env_vars
     @client_secret = data['client_secret']
     data['client_secret'] = nil
 
-    fp = File.open('/Users/stephen/src/gds/inventory-tool/config/new-config.json', 'w')
-    fp.write(JSON.pretty_generate(data))
-    fp.close
+    config_file = File.open('config/google_drive_config.json', 'w')
```

Comment:
If you use the block form of `File.open` and write to the yielded argument you get a `close` guarantee, for free. :sparkler: